spotify_player/src/state/data.rs

```diff
@@ -32,6 +33,7 @@ pub struct AppData {
 pub struct UserData {
     pub user: Option<rspotify_model::PrivateUser>,
     pub playlists: Vec<Playlist>,
+    pub playlist_folder_nodes: Vec<PlaylistFolderNode>,
```

Comment:
can this be defined as a single root node instead?

Reply:
Probably yes. This will require the use of an Option, but in turn will remove the `jq` dependency in the preparation step. Definitely worth a try. 

```diff
- spotifyfolders | jq '.children' > ~/.cache/spotify-player/PlaylistFolders_cache.json
+ spotifyfolders > ~/.cache/spotify-player/PlaylistFolders_cache.json`
```

Reply:
It worked. Now jq is not needed. Thanks for suggestion